tst/testinstall/type.tst

```diff
@@ -0,0 +1,61 @@
+#
+gap> IsFilter(IsFinite);
+true
+gap> IsCategory(IsFinite);
+false
+gap> IsAttribute(IsFinite);
+true
+gap> IsProperty(IsFinite);
+true
+gap> IsOperation(IsFinite);
+true
```

Comment:
Perhaps do this instead (which I find a bit easier to read, esp. as I don't need to scroll around as much; and testing more things later one becomes easier?)
```
gap> test := x -> List([IsFilter, IsCategory, IsAttribute, IsProperty, IsOperation], f -> f(x));;
gap> test(IsFinite)
[ true, false, true, true, true ]
...
```